bin/analyze_hoc_activity

```diff
@@ -154,8 +154,8 @@ def main()
   # Compute the number of hoc events (the grand total and broken down by company and country).
   hoc_country_totals = {}
   hoc_company_totals = {}
-  unique_hoc_events = DB[:forms].where(kind: 'HocSignup2015').group(:name, :email).select(:name, :email, :processed_data, :data)
-  total_hoc_count = unique_hoc_events.count
+  unique_hoc_events = DB[:forms].where(kind: 'HocSignup2015').group(:email, :name).select(:name, :email, :processed_data, :data)
```

Comment:
good point- since the helper methods are now unused, i added a commit to clean this up by removing the helper methods entirely.

this also makes it clear that no database-queries should be done directly from pegasus page-rendering anyway.